### DISCUSSION TITLE
segmentation does not finish on some T1 images

### DISCUSSION THREAD

**Discussion by QingfangLiu** [simnibs/simnibs#426] [7.Creating head models]
Hi, I'm running charm command on subjects' T1 images. Most of them were finished fine, but for two subjects, the mesh file was not created after the segmentation. It looks like the charm code just stopped running. I attached one log file if anyone could help me take a look and give me some pointers. Thank you!

[charm_log.html.zip](https://github.com/user-attachments/files/17075427/charm_log.html.zip)

**Comment by bjornsig** [comment]
It looks like it stops during the meshing. You can try to run just the meshing step for this subject and see if that finishes. You can use the  --mesh flag with charm: charm subject_id --mesh. For the subject in this report file the subject_id is NODEAP_73.
